docs/creating-a-new-adapter.md

```diff
@@ -0,0 +1,77 @@
+# Adapter Basics
```

Comment:
We're making these only viewable on hubot.github.com now. You'll need some front matter like:

```
---
layout: docs
permalink: /docs/creating-a-new-adapter/index.html
---
```